user_guide_src/source/changelogs/v4.4.0.rst

```diff
@@ -121,6 +121,9 @@ Others
 - **Request:** Added ``IncomingRequest::setValidLocales()`` method to set valid locales.
 - **Table:** Added ``Table::setSyncRowsWithHeading()`` method to synchronize row columns with headings. See :ref:`table-sync-rows-with-headings` for details.
 - **Error Handling:** Now you can use :ref:`custom-exception-handlers`.
+- **RedirectException:** can also take an object that implements ResponseInterface as its first argument.
+- **RedirectException:** implements ResponsableInterface
+- **force_https:** no longer terminates the application, but throws a RedirectException.
```

Comment:
`force_https()` is a function, so not in *Others*.
```diff
--- a/user_guide_src/source/changelogs/v4.4.0.rst
+++ b/user_guide_src/source/changelogs/v4.4.0.rst
@@ -102,6 +102,8 @@ Helpers and Functions
 
 - **Array:** Added :php:func:`array_group_by()` helper function to group data
   values together. Supports dot-notation syntax.
+- **Common:** :php:func:`force_https()` no longer terminates the application, but
+  throws a ``RedirectException``.
 
 Others
 ======
@@ -123,7 +125,6 @@ Others
 - **Error Handling:** Now you can use :ref:`custom-exception-handlers`.
 - **RedirectException:** can also take an object that implements ResponseInterface as its first argument.
 - **RedirectException:** implements ResponsableInterface
-- **force_https:** no longer terminates the application, but throws a RedirectException.
 
 Message Changes
 ***************
```

Reply:
Oh thanks for the suggested fix. Apparently I didn't notice this section.